src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -313,6 +313,55 @@ impl ArithmeticOp for DecimalMultiply {
     }
 }
 
+#[derive(Debug)]
+pub struct IntDivideInt;
+
+impl ArithmeticOp for IntDivideInt {
+    type T = Int;
+
+    fn calc(lhs: &Int, rhs: &Int) -> Result<Option<Int>> {
```

Comment:
The implementation is incorrect. Please refer to existing implementation in `int_divide_int`.